pkg/serverstate/statetest/statetest.go

```diff
@@ -33,7 +33,10 @@ type (
 // Test runs a validation test suite for a state implementation. All
 // state implementations should pass this suite with no errors to ensure
 // the correct behavior of the state when Waypoint uses it.
-func Test(t *testing.T, f Factory, rf RestartFactory) {
+// skipTests are function names of tests in the serverstate package to skip
+// (i.e. TestJobCreate_singleton). It cannot skip sub-tests (called by
+// t.Run() inside a top-level test)
+func Test(t *testing.T, f Factory, rf RestartFactory, skipTests []string) {
```

Comment:
How is `skipTests` populated? Could it be populated similar to how the current `tests` slice is?

Reply:
The workflow I envision is that you'd run the tests from a `_test` file in your project, see one that's failing, decide that it's OK from your perspective, and add that test name to the slice where you invoke this `Test` function.

Reply:
👍 for posterity, the `skipTests` slice would be added here: 

-https://github.com/hashicorp/waypoint/blob/d4245103aecefae370587efb92695398e21d1bac/internal/server/boltdbstate/state_test.go#L20

I think we need to add a `nil` or empty slice there, yeah?